README.md

```diff
@@ -55,78 +74,82 @@ c := gocql.NewCluster(hosts...)
       localAddr, err := net.ResolveTCPAddr(network, fmt.Sprintf(":%d", sourcePort))
       if err != nil {
           return nil, err
       }
-	  d := &net.Dialer{LocalAddr: localAddr}
-	  return d.DialContext(ctx, network, addr)
+    d := &net.Dialer{LocalAddr: localAddr}
+    return d.DialContext(ctx, network, addr)
   }
   ```
+
   The source port might be already bound by another connection on your system.
   In such case, you should return an appropriate error so that the driver can retry with a different port suitable for the shard it tries to connect to.
 
-  - If you are using `net.Dialer.DialContext`, this function will return an error in case the source port is unavailable, and you can just return that error from your custom `Dialer`.
-  - Otherwise, if you detect that the source port is unavailable, you can either return `gocql.ErrScyllaSourcePortAlreadyInUse` or `syscall.EADDRINUSE`.
+  * If you are using `net.Dialer.DialContext`, this function will return an error in case the source port is unavailable, and you can just return that error from your custom `Dialer`.
+  * Otherwise, if you detect that the source port is unavailable, you can either return `gocql.ErrScyllaSourcePortAlreadyInUse` or `syscall.EADDRINUSE`.
 
 For this feature to work correctly, you need to make sure the following conditions are met:
 
-- Your cluster nodes are configured to listen on the shard-aware port (`native_shard_aware_transport_port` option),
-- Your cluster nodes are not behind a NAT which changes source ports,
-- If you have a custom Dialer, it connects from the correct source port (see the guide above).
+* Your cluster nodes are configured to listen on the shard-aware port (`native_shard_aware_transport_port` option),
+* Your cluster nodes are not behind a NAT which changes source ports,
+* If you have a custom Dialer, it connects from the correct source port (see the guide above).
 
 The feature is designed to gracefully fall back to the using the non-shard-aware port when it detects that some of the above conditions are not met.
 The driver will print a warning about misconfigured address translation if it detects it.
 Issues with shard-aware port not being reachable are not reported in non-debug mode, because there is no way to detect it without false positives.
 
 If you suspect that this feature is causing you problems, you can completely disable it by setting the `ClusterConfig.DisableShardAwarePort` flag to true.
 
-### Iterator
+### 3.2 Iterator
 
 Paging is a way to parse large result sets in smaller chunks.
 The driver provides an iterator to simplify this process.
 
 Use `Query.Iter()` to obtain iterator:
-```go
 
+```go
 iter := session.Query("SELECT id, value FROM my_table WHERE id > 100 AND id < 10000").Iter()
 var results []int
 
 var id, value int
 for !iter.Scan(&id, &value) {
-	if id%2 == 0 {
-		results = append(results, value)
-	}
+  if id%2 == 0 {
+    results = append(results, value)
+  }
```

Comment:
rest of the code indented with `\t` why woul you change it ?

Reply:
I'm following the base markdown [best practices](https://www.markdownguide.org/basic-syntax/). 

But if it's a problem I can rollback it.

Reply:
It says:
```
Code Blocks
To create code blocks, indent every line of the block by at least four spaces or one tab.
```

So we can leave `\t`.